src/components/Anchor/Anchor.md

```diff
@@ -0,0 +1,7 @@
+A generic `Anchor` component with props for customization
+
+```jsx
+import { Anchor } from '@moneypensionservice/directories';
+
+<Anchor name="andre's page" href="www.andrespage.com" />
+```
```

Comment:
We have a separate ticket for an Anchor component (TP11260) so might be better to leave this to be done in that ticket. I don't think it's a requirement for this component.

Reply:
I attempted this ticket but couldnt go on with it because it would require the component taking nexjs Link component as a prop and that was when i thought we should restrict the library to styles because if we can get our way around this, what about redux and other libraries that the react library will impede. 
So, i created another branch for the footer contact panel but somehow my changes in the other branch still shows here. I felt i must have mistakenly based it on it instead of the library. 
But my intention was to work it out with @andreferreiradlw.

Reply:
> I attempted this ticket but couldnt go on with it because it would require the component taking nexjs Link component as a prop and that was when i thought we should restrict the library to styles because if we can get our way around this, what about redux and other libraries that the react library will impede.
> So, i created another branch for the footer contact panel but somehow my changes in the other branch still shows here. I felt i must have mistakenly based it on it instead of the library.
> But my intention was to work it out with @andreferreiradlw.

@akenzua I'm not sure about the Footer ticket and the issues you are suggesting. We may need to discuss those in relation to that ticket I guess.
I don't quite follow your point: are you saying that you will be updating this branch further to remover references to the Anchor component?